src/QuDiffHHL.jl

```diff
@@ -0,0 +1,102 @@
+export array_qudiff, prepare_init_state, bval, aval
+import QuAlgorithmZoo.hhlsolve
```

Comment:
Have you moved HHL to this repo or do we still need this dep? if we do, you need to add the QuAlgorithmZoo back to make CI pass first.

Reply:
I added QuAlgorithmZoo and included the dep inside `Project.toml` .